src/firefly/java/edu/caltech/ipac/firefly/server/ServerCommandAccess.java

```diff
@@ -141,13 +117,24 @@ private static void initCommand() {
         _cmdMap.put(ServerParams.STATIC_JSON_DATA,           new JsonDataCommands.StaticJsonData());
     }
 
+    public static ServCommand getServCommand(String cmd) {
+        return _cmdMap.get(cmd);
+    }
 
     public static abstract class ServCommand {
         public abstract String doCommand(Map<String, String[]> paramMap) throws Exception;
 
         public boolean getCanCreateJson() { return true; }
     }
 
+    public static abstract class HttpCommand extends ServCommand {
```

Comment:
I appears that ServCommand would be a simpler version HttpCommand.  Maybe it should be an abstract implementation of HttpCommand. The each user of ServCommand could inherit instead of implement.